eventdiscovery-sdk/src/main/res/layout/schedjoules_fragment_event_details.xml

```diff
@@ -56,9 +56,12 @@
                         android:orientation="vertical"
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
-                        android:layout_marginBottom="8dp"
                         android:layout_marginTop="12dp"/>
 
+                <android.support.v4.widget.Space
```

Comment:
This should be hidden by default and only become visible when the ticket button becomes visible too. Otherwise we have to much white space underneath events without ticket option.